RLTSquare/CustomEmailTemplate/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <tab id="rltsquare_config" translate="label" sortOrder="200">
+            <label>RLTSquare</label>
+        </tab>
+        <section id="custom_email_template" translate="label" type="text" sortOrder="42" showInDefault="1"
+                 showInWebsite="1" showInStore="1">
+            <class>separator-top</class>
+            <label>Custom Email Template</label>
+            <tab>rltsquare_config</tab>
+            <resource>RLTSquare_CustomEmailTemplate::custom_email_template</resource>
+            <group id="general" translate="label" sortOrder="92" showInDefault="1" showInWebsite="1" showInStore="1">
+                <label>Custom Email Template</label>
+
+                <field id="enabledisable" translate="label" type="select" sortOrder="5" showInDefault="1"
```

Comment:
Code refactoring - this would need to be refactored, as in why use "enabledisable" for field name when Magento is full fo examples in its code for this and they use "enable"